### DISCUSSION TITLE
Custom Skins Tutorial Help

### DISCUSSION THREAD

**Discussion by Waaddlleess** [Luke100000/minecraft-comes-alive#389] [Q&A]
I followed the tutorial, I have custom skins and the tutorial isn't adding specific clarity. How to connect it to the actual outfit screen? When using the villager editor it is always the default skins and no new ones show up. Is there a way to have pictures in any way? Or perhaps can someone explain step by step in depth?

Thanks for reading and any help that could be provided. I really enjoy the mod and wanted to add on own spin. :)

**Comment by Luke100000** [comment]
To which tutorial are you referring to? I'm not 100% sure if the one on fandom is up to date, I need to verify that, and I also will upload a sample pack soon because I agree, confusion intensifies. Maybe I have time today.

For your problem, make sure the datapack is recognized and loaded. `/datapacks list`.

**Reply by Waaddlleess** [reply]
I used the tutorial here on the github, sorry for the long time answering. A sample pack would be very useful!

**Reply by Khavyer** [reply]
Did u find a solution?
I am trying uploading and adding new skins but they overwritte instead, but not at all from my customized one.
Edit: I already made it work properly. But the new skins don't appear or visualize. Just keep clicking on Next clothe until u see it. Remember to put also the pack on datapack of your world.
